docs/docs/refs-and-the-dom.md

```diff
@@ -73,7 +73,23 @@ class AutoFocusTextInput extends React.Component {
 }
 ```
 
-You may not use the `ref` attribute on functional components because they don't have instances. You can, however, use the `ref` attribute inside the functional component:
+**You may not use the `ref` attribute on functional components** because they don't have instances. In the below code the ref will always be called with `null`:
```

Comment:
Maybe just say "This won't work:". The fact that it's called with `null` is implementation detail, we might not call it at all in the future, and we'll definitely start warning about this. So we don't people to rely on "calling `null`" as the expected behavior.